setup-chainguard-terraform/README.md

```diff
@@ -11,15 +11,14 @@ particular Chainguard environment.  There are two main things this does:
 ```yaml
 - uses: chainguard-dev/actions/setup-chainguard-terraform@main
   with:
-    # environment determines the environment from which to download the chainctl
-    # binary from, it is required and has no default (for now).
-    # Required.
-    environment: cookie-monster
+    # environment determines the environment from which to download the binary
+    # from.
+    # Optional (default is enforce.dev)
+    environment: enforce.dev
     # audience is the identity token audience to use when creating an identity
-    # token to authenticate with Chainguard, it is required and has no default
-    # (for now).
-    # Required.
-    audience: oscar-the-grouch
+    # token to authenticate with Chainguard.
+    # Optional (default is https://issuer.enforce.dev)
+    audience: https://issuer.enforce.dev
```

Comment:
should we remove the https here? like https://github.com/chainguard-dev/actions/pull/204/files#diff-87424462d8e7cecb778565852e1f658a337f0e6abdf8c30fbd813843a70a54d5R21